external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/TopicPartitionComparator.java

```diff
@@ -0,0 +1,21 @@
+package org.apache.storm.kafka.spout;
+
+import org.apache.kafka.common.TopicPartition;
+
+import java.util.Comparator;
+
+/**
+ * Created by liurenjie on 12/7/16.
+ */
+public enum TopicPartitionComparator implements Comparator<TopicPartition> {
```

Comment:
Nit: I don't think there's anything wrong with making this an enum, but it seems conceptually weird to me. Why not just make this a regular class, and then make the field for it in the spout static?